.gitignore

```diff
@@ -183,6 +183,8 @@ node_modules/
 /ajax/libs/jsSHA/release*
 /ajax/libs/postscribe/ersion
 /ajax/libs/react-router/latest/
+/ajax/libs/react-mdl/0.*
+/ajax/libs/react-mdl/1.0.[0-1]
```

Comment:
As you said https://github.com/cdnjs/cdnjs/pull/11878#issuecomment-334129213, I thought "Before" means earlier than the version, but not include the version. 
So I think we don't have to exclude v1.0.*?
Also, these files can be used in web front-end.
What do you think?

Thank you.